metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -349,6 +357,16 @@ public void replay(PartitionChangeRecord record) {
         log.debug("Applied ISR change record: {}", record.toString());
     }
 
+    public void replay(RemoveTopicRecord record) {
```

Comment:
Should we remove deleted topic name from `topicsByName` also?

Reply:
Good catch.  Fixed.